monkey/infection_monkey/post_breach/post_breach_handler.py

```diff
@@ -38,3 +35,10 @@ def config_to_pba_list() -> Sequence[PBA]:
         :return: A list of PBA objects.
         """
         return PBA.get_instances()
+
+    def run_pba(self, pba):
+        try:
+            LOG.debug("Executing PBA: '{}'".format(pba.name))
+            pba.run()
+        except Exception as e:
+            LOG.error("PBA {} failed. Error info: {}".format(pba.name, e))
```

Comment:
Logging doesn't get messed up?

Reply:
No? Just that all PBA logs are not logged together now.

Reply:
What I'm concerned with is the possibility of log showing:
```
Executing PBA X
Output of PBA Y
Errors of PBA Y
```
This might be a bit misleading, but I think we'll manage.